packagemaker/Program.cs

```diff
@@ -10,25 +10,37 @@ namespace plugin.video.vrt.nu.packagemaker
 {
     class Program
     {
+        private const string ZipfileName = "plugin.video.vrt.nu.zip";
```

Comment:
IMO it would be nice if this included a version number (and maybe even a git hash) so a new build is unique and somewhat identifiable.